C-Sharp.sln

```diff
@@ -30,7 +30,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DataStructures.Tests", "Dat
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Utilities", "Utilities\Utilities.csproj", "{3A41157D-296D-4BFC-A34E-91B5ED7F0905}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Utilities.Tests", "Utilities.Tests\Utilities.Tests.csproj", "{3A41157D-296D-4BFC-A34E-91B5ED7F0905}"
```

Comment:
What was changed here?

Reply:
I forgot to exclude this file before commit, sorry. 😔🙇‍♂️
I commit the original value of solution file in [231e002](https://github.com/fuslonflare/C-Sharp/commit/231e002604a36e84e2195a48def8c9ac02b1db7e)